playground-local/tests/local.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { describe, test } from 'vitest'
+import { setup, createPage } from '@nuxt/test-utils/e2e'
+import { expect as playwrightExpect } from '@playwright/test'
+
+const STATUS_AUTHENTICATED = 'authenticated'
+const STATUS_UNAUTHENTICATED = 'unauthenticated'
+
+describe('Local Provider', async () => {
+  await setup({
+    runner: 'vitest',
+    browser: true
+  })
+
+  test('it works', async () => {
```

Comment:
Do we want to improve this naming? What does "it works" mean?

Reply:
What would you propose? I use that test name very often :)

Reply:
Yeah, let's do it